v2/manager.go

```diff
@@ -411,3 +419,50 @@ func (c *Manager) freeze(path string, state State) error {
 		time.Sleep(1 * time.Millisecond)
 	}
 }
+
+func (c *Manager) OOMEventFD() (uintptr, error) {
```

Comment:
I guess while we are working on this, can we go ahead and just make this generic and return all event types.  We can use the `Event` struct to pass the data back to users